src/utilities/time.ts

```diff
@@ -422,6 +422,48 @@ export function convertDurationStringToInterval(durationString: string): string
   return intervalString ? intervalString : '0';
 }
 
+/**
+ * Convert a Postgres UTC string to milliseconds, for rendering purposes.
+ * @param date The date as a string to convert.
+ * @returns The number of milliseconds since epoch start for this date.
+ */
+export function convertUTCtoMs(date: string): number {
```

Comment:
Can we have some unit tests for these new functions?

Reply:
Addressed in [add test for convertUTCtoMs](https://github.com/NASA-AMMOS/aerie-ui/pull/1396/commits/ca8226ddbcf0627a7005fa9468aab4021e965034).